releaser/make_release.py

```diff
@@ -354,10 +354,14 @@ def run_steps(config, steps, steps_funcs):
     for step_func, step_desc in steps_funcs[start:stop]:
         if step_desc:
             print(step_desc + '...', end=' ')
-            step_func(**config)
+
+        config_update = step_func(**config)
```

Comment:
What is the interest of updating the `config` dict after the call to `step_func()`? There is one but I don't see it.

Reply:
so that steps can use information provided/computed in a previous step and not only in set_config.

Reply:
Ok. Thanks for the explanation.